scripts/utils.js

```diff
@@ -0,0 +1,53 @@
+const hre = require('hardhat');
+
+const networks = {
+  mainnets: [
+    'mainnet',
+    'arbitrum',
+    'avalanche',
+    'bsc',
+    'fantom',
+    'gnosis',
+    'metis',
+    'milkomeda',
+    'moonbeam',
+    'moonriver',
+    'optimism',
+    'polygon',
+    'rsk',
+  ],
+  testnets: [
+    'goerli',
+    'sepolia',
+    'avalanche_testnet',
+    'milkomeda_testnet',
+    'fantom_testnet',
+    'moonbase_testnet',
+    'bsc_testnet',
+    'rsk_testnet',
+    'polygon_testnet',
+    'arbitrum_testnet',
+    'optimism_testnet',
+    'gnosis_testnet',
+  ],
+};
```

Comment:
Instead, define two `credetials.json` files. One keeps the testnets and the other keeps the mainnets. Import them together. We detect if a chain is a testnet based on which file it's in. Otherwise this and the credentials file risk falling out of sync.